packages/sdk/CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 All notable changes to the IBM® CICS® Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- Added JVM Endpoint CICS CMCI constant. [#410](https://github.com/zowe/cics-for-zowe-client/pull/410)
+
 ## `6.12.0`
 
 - Added headers as optional parameter to getResource and getCache methods.
 
+
```

Comment:
Not to be picky but I don't know if this has an impact on the Chanagelog file that's released - ie, might have a larger space. Not sure! 

```suggestion
```